packages/connect-explorer-webextension/webpack/build.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+const fetch = require('cross-fetch');
+
+const rootPath = path.join(__dirname, '..');
+const srcPath = path.join(__dirname, '..', '..', 'connect-web');
+const vendorPath = path.join(rootPath, 'vendor');
+
+// TODO: We should not need trezor-connect.js in the extension. It should be loaded from npm
```

Comment:
Yes, it is actually totally unnecessary now since Connect Explorer includes connect in it.

@trezor/connect-explorer is just imported and it is webpack doing the build.

I am now trying to get rid of this script by copying `trezor-content-script.js` with webpack.